python/example/brunel.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+
+import arbor
+import numpy,argparse
+
+'''
+A Brunel network consists of nexc excitatory LIF neurons and ninh inhibitory LIF neurons.
+Each neuron in the network receives in_degree_prop * nexc excitatory connections
+chosen randomly, in_degree_prop * ninh inhibitory connections and next (external) Poisson connections.
+All the connections have the same delay. The strenght of excitatory and Poisson connections is given by
+parameter weight, whereas the strength of inhibitory connections is rel_inh_strength * weight.
+Poisson neurons all spike independently with expected number of spikes given by parameter poiss_lambda.
+Because of the refractory period, the activity is mostly driven by Poisson neurons and
+recurrent connections have a small effect.
+
+Call with parameters, for example:
+./brunel.py -n 400 -m 100 -e 20 -p 0.1 -w 1.2 -d 1 -g 0.5 -l 5 -t 100 -s 1 -G 50 -S 123 -f spikes.txt
+
+'''
+
+# Samples m unique values in interval [start, end) - gid.
+# We exclude gid because we don't want self-loops.
+def sample_subset(gid, start, end, m):
+    gen = numpy.random.default_rng(gid+42)
```

Comment:
The only issue here is that this interface is new in numpy 1.17, and the latest version available in e.g. Debian stable is 1.16.2. The old interface is still preserved under `numpy.random.RandomState`, and we could get 1.16 compatibility with e.g.
```
gen = numpy.random.RandomState(gid+42)
…
val = gen.randint(low=start, high=end)
```

That said though, `Generator.integers` will pick numbers from [low, high) unless `endpoint=True`, and it is by default false, so line 27 should be, IIUC, `val = gen.integers(low=start, high=end)` in the new `numpy.random` interface.